src/controller/sms/get_step_name.js

```diff
@@ -0,0 +1,18 @@
+const steps = require('./steps')
+const debug = require('debug')('sms') // eslint-disable-line
+
+/**
+ * getNextStepName - gets the current step name based on session data
+ *
+ * @param  {object} session session object from request
+ * @return {string} name of current step
+ */
+module.exports = function getStepName(session) {
+  const stepName = Object.keys(steps).reduce((memo, step) => {
+    if (memo) return memo
+    if (!session[step]) return step
```

Comment:
If we did that, we could also have something that hydrates all the step names in the steps array in the session and nulls them out, which would make this check a lot clearer--foreach session.steps return the first where it's null.